spec/models/album_detector_spec.rb

```diff
@@ -8,7 +8,7 @@
 
     let(:photo) do
       FlickrPhoto.create(
-        id: 'flickr photo',
+        id: 'flickr_photo',
```

Comment:
This tweak ensures that our specs pass on both ES 6 & 7, as they handle spaces in IDs differently. See https://github.com/GSA/asis/pull/133